sky/cli.py

```diff
@@ -3063,12 +3063,30 @@ def _output():
                 return
 
         # Show detailed accelerator information
+        accelerator_split = accelerator_str.split(':')
+        if len(accelerator_split) > 2:
+            raise click.UsageError(
+                f'Invalid accelerator string {accelerator_str}. '
+                'Expected format: <accelerator_name>[:<quantity>].')
+        if len(accelerator_split) == 2:
+            name = accelerator_split[0]
+            # Check if quantity is valid
+            try:
+                quantity = int(accelerator_split[1])
+            except ValueError as invalid_quantity:
+                raise click.UsageError(
+                    f'Invalid accelerator quantity {accelerator_split[1]}. '
+                    'Expected an integer.') from invalid_quantity
+        else:
+            name, quantity = accelerator_str, None
+
         result = service_catalog.list_accelerators(gpus_only=True,
-                                                   name_filter=gpu_name,
+                                                   name_filter=name,
+                                                   quantity_filter=quantity,
                                                    region_filter=region,
                                                    clouds=cloud)
         if len(result) == 0:
-            yield f'Resources \'{gpu_name}\' not found. '
+            yield f'Resources \'{name}\', with requested quantity, not found. '
```

Comment:
Can we make this sentence conditional on whether quantity is provided and print the quantity in the output?
```
quantity_str = f' with requested quantity {quantity}' if quantity else ''
...
yield f'Resources \'{name}\'{quantity_str} not found. '
```